package.json

```diff
@@ -15,22 +15,28 @@
   "type": "module",
   "packageManager": "yarn@3.8.7",
   "devDependencies": {
+    "@backstage/catalog-model": "^1.7.3",
```

Comment:
This is the multi-workspace root package.json. We souldn't add this dependencies here.

Can you please revert the `/package.json` and `/yarn.lock` changes and see if you can add this just to your CLI package.